website/components/Navbar.tsx

```
@@ -153,22 +182,22 @@ export default function Navbar() {
                     transition={{ duration: isScrolled ? 0.15 : 0.3, ease: 'easeOut' }}
                 />
                 {/* Logo */}
-                <Link href="/" className="flex items-center gap-2.5 relative z-10 hover:opacity-80 transition-opacity">
+                <Link href="/" className="flex items-center gap-2.5 relative z-10 hover:opacity-80 transition-opacity" onClick={(e) => handleNavClick(e, '/')}>
                     <Image src="/logo.svg" alt="CaptureAI" width={28} height={28} />
                     <span className="text-[15px] font-semibold text-[--color-text]">CaptureAI</span>
                 </Link>
 
                 {/* Desktop nav */}
-                <div className="hidden items-center gap-2 md:flex relative z-10" onMouseLeave={handleMouseLeave}>
+                <div className={`hidden items-center md:flex absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2 z-10 transition-all duration-500 ease-out ${isScrolled ? 'gap-2' : 'gap-4'}`} onMouseLeave={handleMouseLeave}>
```

Biome (2.4.6)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)